src/infrastructure/websocket/GlobalWebSocketService.ts

```
@@ -8,40 +8,55 @@
     const id = `${event}_${++this.idCounter}`;
-    this.events[event].push(listener);
-    this.listenerIds.set(id, event);
+    this.events[event].push({ id, listener });
+    this.listenerIds.set(id, { event, listener });
     return id;
   }
 
   emit(event: string, ...args: any[]) {
     if (this.events[event]) {
-      this.events[event].forEach(listener => listener(...args));
+      this.events[event].forEach(({ listener }) => listener(...args));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)